scripts/basic-workflow-tests.sh

```diff
@@ -296,8 +297,60 @@ test_train() {
 
         ilab model train "${TRAIN_ARGS[@]}"
     fi
+}
+
+test_phased_train() {
+    task Train the model with LAB multi-phase strategy.
+
+    # invariant requirement
```

Comment:
If you have this enforced here, then maybe you don't need to check for the same below in line 456.

Reply:
true, removed